docs/Gemfile.lock

```diff
@@ -51,7 +51,7 @@ GEM
       jekyll-relative-links (= 0.6.1)
       jekyll-remote-theme (= 0.4.1)
       jekyll-sass-converter (= 1.5.2)
-      jekyll-seo-tag (= 2.6.1)
+      jekyll-seo-userType (= 2.6.1)
```

Comment:
Did you accidentally refactor this file?

Reply:
Ah I might've mistakenly done so, thanks!